### PR TITLE
Blog Page

### PR DESCRIPTION
**Pull request by CAS-92**
Create blog page and push header

### CODE REVIEW

File: blog-page.html [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+      </div>
+    </nav>
+    <img
+      src="/assets/images/tiago-rosado-cMG5qjpnsyg-unsplash.jpg"
+      alt="Hero image"
+      class="hero-image"
+    />
+    <p class="hero-text">TravelUp Blog</p>
+  </header>
+
```
Comment (HaydenBS93) [inline blog-page.html:161]: Refer to the original index.html to keep the same layout structure of the header used across all pages

File: blog-page.html [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+      left: 50%; /* Position at the horizontal center */
+      transform: translate(-50%, -50%); /* Center the text */
+      color: white;
+      font-size: 3em;
+      text-align: center;
+      background-color: rgba(0, 0, 0, 0.5); /* Add a semi-transparent background */
+      padding: 20px;
+      border-radius: 10px;
+    }
+
```
Comment (HaydenBS93) [inline blog-page.html:45]: Wont need this once we have used the original index.html header

Don't need any overlay text on the hero image

File: blog-page.html [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+          class="text-travel-orange text-lg font-semibold hover:text-orange-600"
+          >Contact</a
+        >
+      </div>
+    </nav>
+    <img
+      src="/assets/images/tiago-rosado-cMG5qjpnsyg-unsplash.jpg"
+      alt="Hero image"
+      class="hero-image"
+    />
```
Comment (HaydenBS93) [inline blog-page.html:158]: set a max height so it doesn't overflow the screen, other pages will have something similar which you can transfer from

File: blog-page.html [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+    <p class="hero-text">TravelUp Blog</p>
+  </header>
+
+  <div class="blog-entries-container">
+    <div class="blog-entries">
+      <!-- Blog entries -->
+      <!-- Blog entry 1 -->
+      <div class="bg-white rounded-lg shadow-md overflow-hidden blog-entry">
+        <img src="assets/images/carolin-thiergart-p1DPZA4lJuc-unsplash.jpg" alt="Accessories" style="width: 400px;">
```
Comment (HaydenBS93) [inline blog-page.html:167]: check image sizes and use them from the design to implent across all blog posts

File: blog-page.html [hunk 5]
```diff
@@ -0,0 +1,285 @@
...
+          </div>
+        </div>
+      </div>
+
+      <div style="margin-bottom: 20px;"></div>
+
+      <!-- Blog entry 2 -->
+      <div class="bg-white rounded-lg shadow-md overflow-hidden blog-entry">
+        <img src="assets/images/laura-chouette-g2rFQcMADOU-unsplash.jpg" alt="Beauty" style="width: 400px;">
+        <div class="blog-entry-content px-6 py-4">
```
Comment (HaydenBS93) [inline blog-page.html:183]: check image sizes and use them from the design to implent across all blog posts

File: blog-page.html [hunk 6]
```diff
@@ -0,0 +1,285 @@
...
+            <button class="py-4 px-8 bg-white border border-black rounded-lg hover:bg-gray-200 text-travel-orange"><b>READ MORE »</b></button>
+          </div>
+        </div>
+      </div>
+
+      <div style="margin-bottom: 20px;"></div>
+
+      <!-- Blog entry 3 -->
+      <div class="bg-white rounded-lg shadow-md overflow-hidden blog-entry">
+        <img src="assets/images/neom-StbFt6TU1xs-unsplash.jpg" alt="Tech" style="width: 400px;">
```
Comment (HaydenBS93) [inline blog-page.html:197]: check image sizes and use them from the design to implent across all blog posts

File: blog-page.html [hunk 7]
```diff
@@ -0,0 +1,285 @@
...
+          <div>
+            <span class="text-16px font-semibold">Eco-Friendly Travel: Sustainable Accessories for Responsible Explorers</span><button class="bg-blue-500 hover:bg-blue-700 text-black font-bold py-2 px-4 rounded">
+              Read More
+            </button><br>
+            <span></span>
+          </div>
+        </li>
+      </ul>
+    </aside>
+  </div>
```
Comment (HaydenBS93) [inline blog-page.html:247]: Separate these posts with a divider, so maybe contain each popular post in its own container and add a bottom border

File: blog-page.html [hunk 8]
```diff
@@ -0,0 +1,285 @@
...
+      <img src="/assets/icons/youtube.svg" alt="Youtube icon" />
+    </a>
+    <a href="https://www.instagram.com/">
+      <span class="sr-only">Instagram</span>
+      <img src="/assets/icons/instagram.svg" alt="Instagram icon" />
+    </a>
+  </div>
+  <p class="mt-10 text-center text-lg text-black">
+    &copy; 2024 TravelUp, Inc. All rights reserved.
+  </p>
```
Comment (HaydenBS93) [inline blog-page.html:282]: use footer from index.html

File: blog-page.html [hunk 9]
```diff
@@ -0,0 +1,285 @@
...
+  <script src="scripts/scripts.js"></script>
```
Comment (HaydenBS93) [inline blog-page.html:283]: add scroll button code and javascript as per the landing page

File: blog.html [hunk 10]
```diff
@@ -0,0 +1,350 @@
...
+  <div class="blog-entries-container">
+    <div class="blog-entries">
+      <!-- Blog entries -->
+      <!-- Blog entry 1 -->
+      <div class="bg-white rounded-lg shadow-md overflow-hidden blog-entry">
+        <img src="assets/images/carolin-thiergart-p1DPZA4lJuc-unsplash.jpg" alt="Accessories" style="width: 200px; height: 300px;">
+        <div class="blog-entry-content px-6 py-4">
+          <div class="font-bold text-xl mb-2">5 Must-Have Travel Accessories to Elevate Your Adventures: A Comprehensive Guide</div>
+          <p class="text-sm text-gray-600">04/03/2024<span class="opacity-50"></span></p>
+          <p class="mt-4 text-gray-800">Mauris neque quam, fermentum ut nisl vitae, convallis maximus nisl. Sed mattis nunc id lorem euismod placerat. Vivamus porttitor magna enim, ac accumsan tortor cursus at. Phasellus sed ultricies mi non congue ullam corper. Praesent tincidunt sed tellus ut rutrum. Sed vitae justo condimentum, porta lectus vitae, ultricies congue gravida diam non fringilla.</p>
```
Comment (naha92) [inline blog.html:147]: chatgpt job filler text

File: blog.html [hunk 11]
```diff
@@ -0,0 +1,350 @@
...
+          <div class="flex justify-center mt-10">
+            <button class="py-4 px-8 bg-white border border-black rounded-lg hover:bg-gray-200 text-travel-orange"><b>READ MORE »</b></button>
+          </div>
+        </div>
+      </div>
+      <hr style="height: 4px; background-color: orange; border: none;">
```
Comment (naha92) [inline blog.html:153]: not needed now that there is dropshadow

File: blog.html [hunk 12]
```diff
@@ -0,0 +1,350 @@
...
+
+      <div style="margin-bottom: 50px;"></div>
+
+      <!-- Blog entry 2 -->
+      <div class="bg-white rounded-lg shadow-md overflow-hidden blog-entry">
+        <img src="assets/images/laura-chouette-g2rFQcMADOU-unsplash.jpg" alt="Beauty" style="width: 200px; height: 300px;">
+        <div class="blog-entry-content px-6 py-4">
+          <div class="font-bold text-xl mb-2">Skincare Tips for Jet-Setters: How to Keep Your Skin Glowing While Traveling</div>
+          <p class="text-sm text-gray-600">01/11/2023 <span class="opacity-50"></span></p>
+          <p class="mt-4 text-gray-800">Mauris neque quam, fermentum ut nisl vitae, convallis maximus nisl. Sed mattis nunc id lorem euismod placerat. Vivamus porttitor magna enim, ac accumsan tortor cursus at. Phasellus sed ultricies mi non congue ullam corper. Praesent tincidunt sed tellus ut rutrum. Sed vitae justo condimentum, porta lectus vitae, ultricies congue gravida diam non fringilla.</p>
```
Comment (naha92) [inline blog.html:163]: chatgpt job filler text

File: blog.html [hunk 13]
```diff
@@ -0,0 +1,350 @@
...
+          <div class="flex justify-center mt-10">
+            <button class="py-4 px-8 bg-white border border-black rounded-lg hover:bg-gray-200 text-travel-orange"><b>READ MORE »</b></button>
+          </div>
+        </div>
+      </div>
+      <hr style="height: 4px; background-color: orange; border: none;">
```
Comment (naha92) [inline blog.html:169]: not needed now that there is dropshadow

File: blog.html [hunk 14]
```diff
@@ -0,0 +1,350 @@
...
+
+      <div style="margin-bottom: 50px;"></div>
+
+      <!-- Blog entry 3 -->
+      <div class="bg-white rounded-lg shadow-md overflow-hidden blog-entry">
+        <img src="assets/images/nathan-mcdine-7OnrI4rJXcs-unsplash.jpg" alt="Tech" style="width: 200px; height: 300px;">
+        <div class="blog-entry-content px-6 py-4">
+          <div class="font-bold text-xl mb-2">From Drone Photography to 360-Degree Cameras: Capturing Your Travel Adventures in Technicolor</div>
+          <p class="text-sm text-gray-600">21/09/2023<span class="opacity-50"></span></p>
+          <p class="mt-4 text-gray-800">Mauris neque quam, fermentum ut nisl vitae, convallis maximus nisl. Sed mattis nunc id lorem euismod placerat. Vivamus porttitor magna enim, ac accumsan tortor cursus at. Phasellus sed ultricies mi non congue ullam corper. Praesent tincidunt sed tellus ut rutrum. Sed vitae justo condimentum, porta lectus vitae, ultricies congue gravida diam non fringilla.</p>
```
Comment (naha92) [inline blog.html:179]: chatgpt job filler text

File: blog.html [hunk 15]
```diff
@@ -0,0 +1,350 @@
...
+          <div class="flex justify-center mt-10">
+            <button class="py-4 px-8 bg-white border border-black rounded-lg hover:bg-gray-200 text-travel-orange"><b>READ MORE »</b></button>
+          </div> 
+        </div>
+      </div>
+      <hr style="height: 4px; background-color: orange; border: none;">
```
Comment (naha92) [inline blog.html:185]: not needed now that there is dropshadow

File: blog.html [hunk 16]
```diff
@@ -0,0 +1,350 @@
...
+          </div>
+        </li>
+      </ul>      
+    </aside>
+    </div>
+    <br>
+    <br>
+    <div class="button-container">
+      <button class="py-4 px-8 bg-white border border-black rounded-lg hover:bg-gray-200 text-travel-orange"><b>Load More Posts »</b></button>
+    </div>
```
Comment (naha92) [inline blog.html:236]: <br> not best practice, better to use margins top/bottom to create space

<aside class="popular-posts"> would recommend not using this, check newsletter section on index.html or also how you have made the blog entries. Will remove having to make responsive you .popular.post styling

This section along with the blog posts should be in a 2 column grid that turns into a 1 column grid on small screens, check main branch code for tips on how to make responsive (All section are responsive on main now so can copy/paste what you need)

Centre images with text

Comment (naha92) [reply blog.html:236]: hasn't let me include the code in here, for the not best practice that is using `<br>`
wouldn't recomment using `<aside class="popular-posts">`

File: blog.html [hunk 17]
```diff
@@ -0,0 +1,350 @@
...
+     <!-- Scroll Button -->
+     <button
+     onclick="topFunction()"
+     id="my-btn"
+     title="Go to top"
+     class="hidden fixed bottom-20 right-10 z-50 bg-travel-orange text-white font-bold py-2 px-4 rounded-lg hover:bg-orange-600"
+   >
+     Top
```
Comment (naha92) [inline blog.html:244]: Button isn't working for me, does it work for you?

File: blog.html [hunk 18]
```diff
@@ -0,0 +1,350 @@
...
+      <!-- Blog entries -->
+      <!-- Blog entry 1 -->
+      <div class="bg-white rounded-lg shadow-md overflow-hidden blog-entry">
+        <img src="assets/images/carolin-thiergart-p1DPZA4lJuc-unsplash.jpg" alt="Accessories" style="width: 200px; height: 300px;">
+        <div class="blog-entry-content px-6 py-4">
+          <div class="font-bold text-xl mb-2">5 Must-Have Travel Accessories to Elevate Your Adventures: A Comprehensive Guide</div>
+          <p class="text-sm text-gray-600">04/03/2024<span class="opacity-50"></span></p>
+          <p class="mt-4 text-gray-800">Mauris neque quam, fermentum ut nisl vitae, convallis maximus nisl. Sed mattis nunc id lorem euismod placerat. Vivamus porttitor magna enim, ac accumsan tortor cursus at. Phasellus sed ultricies mi non congue ullam corper. Praesent tincidunt sed tellus ut rutrum. Sed vitae justo condimentum, porta lectus vitae, ultricies congue gravida diam non fringilla.</p>
+          <div class="flex justify-center mt-10">
+            <button class="py-4 px-8 bg-white border border-black rounded-lg hover:bg-gray-200 text-travel-orange"><b>READ MORE »</b></button>
```
Comment (naha92) [inline blog.html:149]: Refer to index.html to keep consistency for buttons

File: blog.html [hunk 19]
```diff
@@ -0,0 +1,350 @@
...
+
+      <!-- Blog entry 2 -->
+      <div class="bg-white rounded-lg shadow-md overflow-hidden blog-entry">
+        <img src="assets/images/laura-chouette-g2rFQcMADOU-unsplash.jpg" alt="Beauty" style="width: 200px; height: 300px;">
+        <div class="blog-entry-content px-6 py-4">
+          <div class="font-bold text-xl mb-2">Skincare Tips for Jet-Setters: How to Keep Your Skin Glowing While Traveling</div>
+          <p class="text-sm text-gray-600">01/11/2023 <span class="opacity-50"></span></p>
+          <p class="mt-4 text-gray-800">Mauris neque quam, fermentum ut nisl vitae, convallis maximus nisl. Sed mattis nunc id lorem euismod placerat. Vivamus porttitor magna enim, ac accumsan tortor cursus at. Phasellus sed ultricies mi non congue ullam corper. Praesent tincidunt sed tellus ut rutrum. Sed vitae justo condimentum, porta lectus vitae, ultricies congue gravida diam non fringilla.</p>
+          <div class="flex justify-center mt-10">
+            <button class="py-4 px-8 bg-white border border-black rounded-lg hover:bg-gray-200 text-travel-orange"><b>READ MORE »</b></button>
```
Comment (naha92) [inline blog.html:165]: Refer to index.html to keep consistency for buttons

File: blog.html [hunk 20]
```diff
@@ -0,0 +1,350 @@
...
+
+      <!-- Blog entry 3 -->
+      <div class="bg-white rounded-lg shadow-md overflow-hidden blog-entry">
+        <img src="assets/images/nathan-mcdine-7OnrI4rJXcs-unsplash.jpg" alt="Tech" style="width: 200px; height: 300px;">
+        <div class="blog-entry-content px-6 py-4">
+          <div class="font-bold text-xl mb-2">From Drone Photography to 360-Degree Cameras: Capturing Your Travel Adventures in Technicolor</div>
+          <p class="text-sm text-gray-600">21/09/2023<span class="opacity-50"></span></p>
+          <p class="mt-4 text-gray-800">Mauris neque quam, fermentum ut nisl vitae, convallis maximus nisl. Sed mattis nunc id lorem euismod placerat. Vivamus porttitor magna enim, ac accumsan tortor cursus at. Phasellus sed ultricies mi non congue ullam corper. Praesent tincidunt sed tellus ut rutrum. Sed vitae justo condimentum, porta lectus vitae, ultricies congue gravida diam non fringilla.</p>
+          <div class="flex justify-center mt-10">
+            <button class="py-4 px-8 bg-white border border-black rounded-lg hover:bg-gray-200 text-travel-orange"><b>READ MORE »</b></button>
```
Comment (naha92) [inline blog.html:181]: Refer to index.html to keep consistency for buttons

File: blog.html [hunk 21]
```diff
@@ -0,0 +1,350 @@
...
+            <br>
+          </div>
+        </li>
+      </ul>      
+    </aside>
+    </div>
+    <br>
+    <br>
+    <div class="button-container">
+      <button class="py-4 px-8 bg-white border border-black rounded-lg hover:bg-gray-200 text-travel-orange"><b>Load More Posts »</b></button>
```
Comment (naha92) [inline blog.html:235]: Refer to index.html to keep consistency for buttons

File: blog.html [hunk 22]
```diff
@@ -0,0 +1,529 @@
...
+            <div class="flex justify-center mt-10">
+              <button
+                type="button"
+                class="px-4 py-2 text-white text-base font-medium rounded-lg bg-travel-orange hover:bg-orange-600"
+              >
+                Read More
+              </button>
+            </div>
+          </div>
+        </div>
```
Comment (HaydenBS93) [inline blog.html:276]: Use the same format for the other posts and reduce text where it needs .

File: blog.html [hunk 23]
```diff
@@ -0,0 +1,529 @@
...
+
+        <div style="margin-bottom: 50px"></div>
+
+        <!-- Blog entry 2 -->
```
Comment (HaydenBS93) [inline blog.html:280]: Remove

File: blog.html [hunk 24]
```diff
@@ -0,0 +1,529 @@
...
+              >
+                Read More
+              </button>
+            </div>
+          </div>
+        </div>
+
+        <div style="margin-bottom: 50px"></div>
+
+        <!-- Blog entry 3 -->
```
Comment (HaydenBS93) [inline blog.html:323]: Remove all whitespace

File: blog.html [hunk 25]
```diff
@@ -0,0 +1,529 @@
...
+              >
+                Read More
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="popular-post">
+        <aside class="popular-posts">
+          <!-- Popular Posts section -->
```
Comment (HaydenBS93) [inline blog.html:366]: Remove

File: blog.html [hunk 26]
```diff
@@ -0,0 +1,529 @@
...
+    <div class="button-container">
+      <button
+        type="button"
+        class="px-4 py-2 text-white text-base font-medium rounded-lg bg-travel-orange hover:bg-orange-600"
+      >
+        Load More Posts
+      </button>
+    </div>
+
+    <!-- Scroll Button -->
```
Comment (HaydenBS93) [inline blog.html:441]: Remove

File: blog.html [hunk 27]
```diff
@@ -0,0 +1,529 @@
...
+    <button
+      onclick="topFunction()"
+      id="my-btn"
```
Comment (HaydenBS93) [inline blog.html:444]: Check this against your script file, the ID isnt the same, needs to be as per this line

File: blog.html [hunk 28]
```diff
@@ -0,0 +1,529 @@
...
+    >
+      Top
+    </button>
+
+    <!-- Cookie Banner -->
+    <div
+      id="cookie-banner"
+      class="fixed inset-x-0 z-10 bottom-0 flex flex-col justify-between gap-x-8 gap-y-4 p-4 bg-white border-t border-t-travel-orange md:flex-row md:items-center lg:px-8 opacity-0 transition-opacity duration-500 ease-in-out"
+    >
+      <!-- Your existing cookie banner code goes here -->
```
Comment (HaydenBS93) [inline blog.html:456]: Copy over the cookie banner from our home page

**Review by HaydenBS93 — APPROVED** [review]

**Pull request merged**